lib/hanami/action.rb

```diff
@@ -284,7 +288,15 @@ def self.prepend_after(...)
     #   # When called with "application/json" => 200
     #   # When called with "application/xml"  => 415
     def self.accept(*formats)
-      config.accepted_formats = *formats
+      config.accepted_formats = formats
+    end
```

Comment:
Should this really override something that was possibly already set?

Reply:
@solnic that's a good question!

Seems like this matches the behaviour from 1.3.x (and I presume all the prior versions), in that calling `accept` ([see the 1.3.x implementation here](https://github.com/hanami/controller/blob/2496ac797237d7320b96048a3c9e3fbebb00532c/lib/hanami/action/mime.rb#L166)) will replace any other previous values ([see where that happens in 1.3.x](https://github.com/hanami/controller/blob/2496ac797237d7320b96048a3c9e3fbebb00532c/lib/hanami/controller/configuration.rb#L446-L448)).

Given this is pre-established behaviour, I'm inclined to leave it as it is, at least for the sake of this PR.

Reply:
I'll merge this in for now, since it's not changing previous behaviour. We can still adjust this if we like through a future PR.